eclipse-scout-core/src/tree/TreeAdapter.ts

```diff
@@ -319,6 +335,18 @@ export class TreeAdapter extends ModelAdapter {
     objects.replacePrototypeFunction(Tree, '_createTreeNode', TreeAdapter._createTreeNodeRemote, true);
     objects.replacePrototypeFunction(Tree, '_updateMarkChildrenChecked', TreeAdapter._updateMarkChildrenCheckedRemote, true);
   }
+
+  static isRemote(node: AdapterTreeNode, includeHybrid?: boolean): boolean {
+    if (node?.remote) {
+      return true;
+    }
+    return includeHybrid ? !!node.__hybrid : false;
+  }
 }
 
 App.addListener('bootstrap', TreeAdapter.modifyTreePrototype);
+
+export interface AdapterTreeNode extends TreeNode {
+  remote?: boolean;
```

Comment:
I think we already discussed this, but I forgot why this property does not use the double underscore prefix.

Reply:
The property exists already on the `Page` and was moved to the `TreeNode`. Therefore, it was not renamed to minimize code migration.